packages/code-studio/src/styleguide/StyleGuideRoot.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect } from 'react';
+import { Provider } from 'react-redux';
+import 'fira';
+import '@deephaven/components/scss/BaseStyleSheet.scss';
+import { MonacoUtils } from '@deephaven/console';
+import { store } from '@deephaven/redux';
+import MonacoWorker from 'monaco-editor/esm/vs/editor/editor.worker?worker';
+import DownloadServiceWorkerUtils from '../DownloadServiceWorkerUtils';
+import { unregister } from '../serviceWorker';
+import StyleGuideInit from './StyleGuideInit';
+
+export function StyleGuideRoot() {
+  useEffect(() => {
+    unregister();
+    DownloadServiceWorkerUtils.registerOnLoaded();
+    MonacoUtils.init({ getWorker: () => new MonacoWorker() });
+
+    // disable annoying dnd-react warnings
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    window['__react-beautiful-dnd-disable-dev-warnings'] = true;
+  }, []);
```

Comment:
This probably doesn't have to be in a `useEffect` really since the root takes no params and has no state, so it shouldn't re-render

Reply:
True... should change all the other `*Root` files as well. I did put it in a `useEffect` as I think that's best practices (if you added a prop later or something), but at the same time - we want these to run before the child renders, and right now the `useEffect` fires _after_ this render returns. So I think more correct to do as you suggest.